web/blog/2023-02-21-junior-dev-misconceptions.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "The Most Common Misconceptions Amongst Junior Developers"
+authors: [vinny]
+image: /img/juniors-misconceptions.png
+tags: [Junior Developers, Tech Career, Reddit, WebDev]
+---
+
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+import InBlogCta from './components/InBlogCta';
+import ImgWithCaption from './components/ImgWithCaption'
+
+<br/>
+
+> *High code quality only indirectly affects users. The main purpose is to keep development velocity high which benefits all stakeholders* 
+> &nbsp; — **zoechi**
+
+<br/>
+
+We recently asked the web dev community on [Reddit.com](https://www.reddit.com/r/webdev/comments/112im2m/senior_devs_what_are_the_most_damaging/) what the most common misconceptions are amongst junior developers, and we got a ton of great responses -- more than 270 to be exact.
+
+Because there was so much to discuss, Matija and I decided to summarize the replies and give our own opinions in a longer-form YouTube video, which you can watch below. 
+
+*You can also continue reading further for a summary of the main concepts.*
+
+<iframe width="100%" height="400" src="https://www.youtube.com/embed/eermNn9VhOA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+<!--truncate-->
+<p></p>
+
+## The Most Common Themes
+
+Among the responses were lots of great, specific examples, but we noticed a lot of common themes within them:
+
+- **Code Quality**
+- **Managing Time & Expectations**
+- **Effective Communication & Teamwork**
+
+These seemed to be the topics senior devs had the most to say about. And it makes sense -- these are the things that, when you get to the core of the issues, can make or break almost *any* career. 
+
+It was also interesting to see that the top replies were issues that encompassed all of these themes. For example, take the top-voted reply:
+
+<p align='center'>
```

Comment:
We actually have an `ImageWithCaption` component you can use here, so you don't have to put all this boilerplate for every image (plus easier to maintain). Check any of my blog posts to see how to import and use it.